requirements.in

```diff
@@ -23,7 +23,7 @@ scrapy-querycleaner
 scrapy-splitvariants
 
 # required by Monitoring addon
-spidermon[monitoring,validation]
+spidermon[monitoring]
```

Comment:
Why are we removing `validation`?

Reply:
@elacuesta It's been [removed](https://github.com/scrapinghub/spidermon/pull/399/files#diff-e285c1b9901cd202e89d9f37e3249a6fee524a65428895e1a45fd062cc0f9acaL18) in the latest release, please see https://github.com/scrapinghub/spidermon/pull/399 for details.